content/release-10-17-0/whatsnew-10-17-0.md

```diff
@@ -68,3 +68,13 @@ This feature is useful, for example, if there is a security issue with the token
 In the **User** page, at the right of the top menu bar, click **Revoke tokens** to revoke all session tokens immediately.
 
 ![User token revocation](/images/release-notes/admin-users-list.png)
+
+### OEE App
+
+An updated version of the Cumulocity IoT OEE application is now available. 
+
+This version adds support for defining one-off and repeating work Shifts on a per-location basis. Doing so will allow calculation of OEE (Overall Equipment Efficiency), Availability, Performance, Quality and various other metrics over these timeframes. 
+
+![Admin shift plan configuration](/images/release-notes/oee-admin-shift-plans.png)
+
+For details, refer to the OEE Guide.
```

Comment:
Add link

Reply:
Done, although I couldn't find a suitable shortcode for the release as per previous URLs. I'm sure there probably is one, let me know if I missed it.